ansible_runner/runner_config.py

```diff
@@ -178,6 +180,18 @@ def prepare(self):
             self.env['ANSIBLE_STDOUT_CALLBACK'] = 'minimal'
         else:
             self.env['ANSIBLE_STDOUT_CALLBACK'] = 'awx_display'
+
+            if self.resource_profiling:
+                callback_whitelist = os.environ.get('ANSIBLE_CALLBACK_WHITELIST', '').strip()
+                self.env['ANSIBLE_CALLBACK_WHITELIST'] = callback_whitelist + (',' if len(callback_whitelist) else '') + 'cgroup_perf_recap'
+                self.env['CGROUP_CONTROL_GROUP'] = '{}/{}'.format(self.resource_profiling_base_cgroup, self.ident)
+                self.env['CGROUP_OUTPUT_DIR'] = self.private_data_dir + '/profiling_data'
```

Comment:
might be nice to wrap this in `os.path.normpath()` just in case an extra trailing `/` sneaks into `private_data_dir` somehow.